phpstan.neon

```diff
@@ -69,3 +69,6 @@ parameters:
 
   # This really has no type in root resolvers
   - '#Method .* has parameter \$root with no typehint specified\.#'
+
+  # PhpStan cannot handle higher order collections
```

Comment:
I would rather not use them in that case, given there is a perfectly fine alternative that preserves type safety.